src/MobileUI/ViewModels/QuizDetailsViewModel.cs

```diff
@@ -16,31 +17,42 @@ public class QuizDetailsViewModel : BaseViewModel
 
         public ObservableCollection<QuestionResultDto> Results { get; set; } = new ObservableCollection<QuestionResultDto>();
 
-        public ICommand ButtonCommand { get; set; }
+        [ObservableProperty]
+        private ICommand _buttonCommand;
```

Comment:
We can do one better here. https://learn.microsoft.com/en-us/dotnet/communitytoolkit/mvvm/relaycommand. But I think we can open a separate issue and PR for that. 

For now, revert these from the `ICommand`s.

Reply:
Done - reverted the ICommand code.